src/engraving/dom/measure.cpp

```diff
@@ -3132,6 +3133,29 @@ Fraction Measure::computeTicks()
     return minTick;
 }
 
+//---------------------------------------------------------
+//   anacrusisOffset
+//      determine if measure is anacrusis
+//      and return tick offset relative to measure end
+//---------------------------------------------------------
+
+Fraction Measure::anacrusisOffset() const
+{
+    Fraction offset = Fraction(0, 1);
+    const MeasureBase* pm = prev();
+    ElementType pt = pm ? pm->type() : ElementType::INVALID;
+
+    if (irregular() || !pm
+        || pm->lineBreak() || pm->pageBreak() || pm->sectionBreak()
+        || pt == ElementType::VBOX || pt == ElementType::HBOX
+        || pt == ElementType::FBOX || pt == ElementType::TBOX) {
+        if (timesig() - ticks() > Fraction(0, 1)) {
```

Comment:
Shouldn't `Measure::isAnacrusis` be updated to use this same condition?

Reply:
> Shouldn't `Measure::isAnacrusis` be updated to use this same condition?

I was afraid that might have unwanted consequences, but I'll give that a try

> Also, do you know what's the relation/difference between `Measure::timesig()` and `score()->sigmap()->timesig(tick().ticks()).nominal()`?

As I recall, `Measure::timesig()` returns a `Fraction` value and the second method returns a `TimeSigFrac` value (which contains more detailed information needed to compute the groupings.